backend/src/main/java/com/team4/backend/exception/ForbiddenActionException.java

```
@@ -0,0 +1,11 @@
+package com.team4.backend.exception;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
```

Missing the @ResponseStatus annotation

---

I'm following our convention, we return a responseStatusException already so no need, we will only repeat ourself

---

If we follow our conventions, the service should throw the exception